src/alephbet.coffee

```diff
@@ -1,9 +1,10 @@
-utils = require('./utils.js.coffee')
-Storage = require('./storage.js.coffee')
-adapters = require('./adapters.js.coffee')
+utils = require('./utils')
+adapters = require('./adapters')
+options = require('./options')
+LocalStorageAdapter = require('./local_storage_adapter')
 
 class AlephBet
-  @options = {debug: false}
+  @options = options
```

Comment:
I think this is only assigned to be able to modify it from the outside like `Alephbet.option.debug = true`.